server/sonar-web/src/main/js/apps/component-measures/components/MeasureContent.js

```diff
@@ -210,7 +212,7 @@ export default class MeasureContent extends React.PureComponent {
     }
   };
 
-  onSelectComponent = (componentKey: string) => this.setState({ selected: componentKey });
+  onSelectComponent = (componentKey /*: string */) => this.setState({ selected: componentKey });
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)